recipes/virtual-libjpeg/all/conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conans import ConanFile, tools
+
+
+class VirtualjpegConan(ConanFile):
+    name = "virtual-libjpeg"
+    description = "Virtual package to choose jpeg encoder/decoder implementation in final package."
+    url = "https://github.com/conan-io/conan-center-index"
+
+    options = {
+        "implementation": ["libjpeg", "libjpeg-turbo"],
```

Comment:
When I first suggested a virtual package for OpenGL, I used the term `provider` instead, which I would personally like better than `implementation`.

What do you all think?

Reply:
good, and that match to "provides" attribute introduce in conan 1.29.0